lib/tasks/work_tasks.rake

```diff
@@ -25,4 +25,14 @@ namespace :work do
     end
   end
 
+  # AO3-6065
+  desc "Add 'Not Rated' to works missing a rating"
```

Comment:
Since this is intended to be a one-time task, it would be more at home in `lib/tasks/after_tasks.rake`.

It also needs some tests, which can be added to `spec/lib/tasks/after_tasks.rake_spec.rb`.